### ISSUE TITLE
graph traversal config: collection _graphs does not exist?! (Aardvark JS Shell)

### ISSUE THREAD

**Issue by CoDEmanX** [arangodb/arangodb#1434]
I tried to write a traverser in the JS shell, but it gave me an error about collection `_graphs` being not available. It does exist however, and if I do `db._collections("_graphs")`, it won't throw that error anymore:
```
JSH> var traversal = require("org/arangodb/graph/traversal"); 
...>
JSH> var config = { 
...>  datasource: traversal.generalGraphDatasourceFactory("watchlist"), 
...>  strategy: "depthfirst", 
...>  order: "preorder", 
...>  filter: traversal.visitAllFilter, 
...>  expander: traversal.inboundExpander, 
...>  visitor: traversal.countingVisitor 
...>};

collection _graphs does not exist //<<< ERROR


JSH> db._collection("_graphs")
==> [ArangoCollection 2195904, "_graphs" (type document, status loaded)] //<<< IS DOES EXIST!

JSH> var config = { 
...>  datasource: traversal.generalGraphDatasourceFactory("watchlist"), 
...>  strategy: "depthfirst", 
...>  order: "preorder", 
...>  filter: traversal.visitAllFilter, 
...>  expander: traversal.inboundExpander, 
...>  visitor: traversal.countingVisitor 
...>};
JSH> //<<< NO MORE ERROR?!
```
Using latest ArangoDB 2.6.4 on Windows 8.

**Comment by CoDEmanX** [comment]
Another example, but it does not resolve by accessing the "missing" collection:
```
JSH> db
==> [object ArangoDatabase "_system"]

JSH> db._collection("users")
==> [ArangoCollection 113029709242, "users" (type document, status loaded)]

JSH> db.users
==> [ArangoCollection 113029709242, "users" (type document, status loaded)]

JSH> graph._fromVertex("users/sameen")
collection does not exist: users
```

**Comment by jsteemann** [comment]
The initially reported bug (*collection not found*) has been fixed in 2.6 and devel.
The second issue (`fromVertex()` returning an unintuitive error message) has been fixed in devel. The latter is because the return code of the function changed in case of this error.
